src/meshdb/settings.py

```diff
@@ -213,3 +215,38 @@
 }
 
 HOOK_CUSTOM_MODEL = "meshapi_hooks.CelerySerializerHook"
+
+SPECTACULAR_SETTINGS = {
+    "TITLE": "MeshDB Data API",
+    "VERSION": "1.0.0",
+    "SERVE_INCLUDE_SCHEMA": False,
+    "TAGS": [
+        {"name": "API Status", "description": "Meta-endpoint to indicate API status"},
+        {"name": "Members", "description": "Members of the mesh and their contact details"},
+        {"name": "Buildings", "description": "Buildings on the mesh and their location data"},
+        {
+            "name": "Installs",
+            "description": "Installs, one corresponding to each household on the mesh or wishing to join",
+        },
+        {"name": "Links", "description": "Links between buildings"},
+        {"name": "Sectors", "description": 'Special antennas with broad coverage of a radial "slice" of land area'},
+        {"name": "Geographic & KML Data", "description": "Endpoints for geographic and KML data export"},
+        {
+            "name": "Website Map Data",
+            "description": "Endpoints used to power the nycmesh.net website map. "
+            "Uses a legacy data format, not recommended for new applications",
```

Comment:
Can we emphasize "NOT RECOMMENDED FOR NEW APPLICATIONS" somehow?

Reply:
Done, but I don't feel great about it. See above